src/app/components/character-list/character-list.component.spec.ts

```diff
@@ -1,14 +1,14 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { CharacterListComponent } from './character-list.component';
+import  Character  from '../../models/character';
```

Comment:
Should be import { Character } from '../../models/character';

Reply:
Good catch.

Reply:
Import from interface, not class. Import should be correct without brackets